scripts/india_census/primary_religion_data/IndiaCensus2011_Primary_Abstract_Religion.tmcf

```diff
@@ -0,0 +1,10 @@
+Node: E:IndiaCensus2011_Primary_Abstract_Religion->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: C:IndiaCensus2011_Primary_Abstract_Religion->StatisticalVariable
+observationDate: C:IndiaCensus2011_Primary_Abstract_Religion->Year
+observationAbout: E:IndiaCensus2011_Primary_Abstract_Religion->E1
+value: C:IndiaCensus2011_Primary_Abstract_Religion->Value
+
+Node: E:IndiaCensus2011_Primary_Abstract_Religion->E1
+typeOf: schema:Place
+indianCensusAreaCode2011: C:IndiaCensus2011_Primary_Abstract_Religion->census_location_id
```

Comment:
I wonder what here makes it religion specific? Can this be in common?

Reply:
Nothing. It can be common. I have used the dataset name. Do you want me to change it everywhere?